packages/table/src/react/index.tsx

```diff
@@ -121,10 +45,20 @@ const Table = React.forwardRef<HTMLTableElement, TableProps>((props, ref) => {
   return (
     <Container
       ref={containerRef}
-      {...styles.container(themeName, { scrollable })}
+      className={classNames(
+        className,
```

Comment:
would compose last.

there are a few more, so I'm going to stop marking these.